portfolio/src/main/webapp/script.js

```diff
@@ -239,80 +281,38 @@ function addCityMarker(map, lat, lng, title, description) {
   });
 }
 
-/** Creates New York map with place markers. */
+/** Creates a map of New York with markers for significant locations in my
+ *      life.
+*/
 function createNewyorkMap() {
   const map = new google.maps.Map(
       document.getElementById('newyork-map'),
       {center: {lat: 40.747345, lng: -73.988955}, zoom: 12});
   document.getElementById("newyork-map").style.display = "block";
-
-  addLandmark(
-      map, 40.717839, -74.013735, 'Stuyvesant High School',
-      'I attended Stuyvesant High School from 2014-2018');
-
-  addLandmark(
-      map, 40.704545, -74.009393, 'Financial District',
-      'I lived in the financial district for almost a year from 2014-2015');
-
-  addLandmark(
-      map, 40.767702, -73.984881, 'Hell\'s Kitchen',
-      'I\'ve live in Hell\'s Kitchen since 2015');
-
-  addLandmark(
-      map, 40.718845, -74.034609, 'Tradeweb',
-      'I interned at Tradeweb in the summer of 2019');
-
-  addLandmark(
-      map, 40.749886, -73.993602, 'Madison Square Garden',
-      'The world famous Madison Square Garden, my favorite concert venue');
-
-  addLandmark(
-      map, 40.780943, -73.966083, 'The Great Lawn',
-      'I love to hang out on the Great Lawn during the summer with friends');
-
-  addLandmark(
-      map, 40.741531, -73.988014, 'Shake Shack',
-      'Shake Shack - my favorite burgers in the world!');
-
-  addLandmark(
-      map, 40.734906, -73.988437, 'Irving Plaza',
-      'Irving plaza is my favorite small concert venue!');
+  
+  for (place of NEWYORK_PLACES) {
+    addMarkerWithInfoWindow(map, place.lat, place.lng, place.title, place.description);
+  }
 }
 
-/** Creates Los Angeles map with place markers. */
+
+/** Creates a map of Los Angeles with markers for significant locations in my
+ *      life.
```

Comment:
Would this fit on one line?

Reply:
No, it goes just over the 80 character line limit.